### PR TITLE
Sprint 6

### PR DESCRIPTION
**Pull request by CharAznable1138**
This pull request contains all the changes I've made since November 5th, 2020.

### CODE REVIEW

File: Assets/Scripts/UI Scripts/HintTextManager.cs [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+    private GameObject player;
+    private PlayerController playerController;
+
+    private bool playerHasSeenMoveHint = false;
+    private bool playerHasSeenShootHint = false;
+    private bool playerHasSeenJumpHint = false;
+    private bool playerHasSeenLockHint = false;
+    private bool playerHasSeenAimHint = false;
+    private enum HintType
```
Comment (fatjosephina) [inline Assets/Scripts/UI Scripts/HintTextManager.cs:49]: I wish that I had some suggestions as to how to do this better, but sadly I don't. However, I will say that your code in general is much less messy than mine, if that gives you any encouragement haha. Only thing I would say is don't forget to add summaries and tool tips before the final release!

**Review by fatjosephina — COMMENTED** [review]
Your hierarchy looks pretty good, all the changes seem to be working, and your code structure seems pretty solid to me. I like what you've done with this project. I commented this on your code already, but the only thing I would look out for it to make sure that summaries and tool tips get added before the final version. Luckily, that is over a month away but it's just something to keep in the back of your mind.

**Comment by fatjosephina** [comment]
As everything looks good, I will be merging this now. But, I'll leave deleting it up to you. Good work.

File: Assets/Scripts/UI Scripts/HintTextManager.cs (hunk 1 above)
Comment (dantogno) [reply Assets/Scripts/UI Scripts/HintTextManager.cs:49]: You could create a hint  text class with a property HasBeenSeen

**Pull request merged**